utils/model_uploader/model_autotracing.py

```diff
@@ -428,12 +449,15 @@ def main(
     :type pooling_mode: string
     :param model_description: Model description input
     :type model_description: string
+    :param third_party_copyrights_statements: Statements text for non Apache-2.0 licensed third party model. Should be put in the final artifact.
```

Comment:
are we sure that for any kind non Apahe-2.0 licenses we need this copy right statements? Or this is only for MIT licenses?